Add tables weight-formats and pre-post-processing (#43)
bioimage-io/configuration: supported_formats_and_operations.md

> # Weight Formats

Reviewer: ```suggestion
# Weights Formats
```
as in Readme.md

> The supported weight formats and their additional fields:

Reviewer: ```suggestion
The supported weights formats and their additional fields:
```

- `keras_hdf5`: A hdf5 file containing weights for Keras.
- `pytorch_script`: A torchscript file.
- `pytorch_state_dict`: A file containg the state dict of a pytorch model.
- `tensorflow_js`: A text JSON file named model.json, which carries the topology and reference to the weights files, used by tensorflow.js.
- `tensorflow_saved_model_bundle`: A zip file containing a `pb` file and `variables` folder. Additional fields are
  - `tag`

>   - `tensorflow-version`

Reviewer: ```suggestion
  - `tensorflow_version`
```
snake case for consistency (and valid python identifiers)

## Consumers

Which consumer software supports which format?

> | `weight_format`       | ilastik | deepImageJ | Fiji |

Reviewer: ```suggestion
| `weights_format`       | ilastik | deepImageJ | Fiji |
```

# Weight Formats

The supported weight formats and their additional fields:

> - `keras_hdf5`: 

Reviewer 2: A hdf5 file containing weights for Keras.

- `pytorch_script`: A torchscript file.
- `pytorch_state_dict`: A file containg the state dict of a pytorch model.

> - `tensorflow_js`: A zip file containing a json file and a binary weights file.

Reviewer 2: Well, the files for tensorflow_js is not a zip file as we discussed.
```suggestion
- `tensorflow_js`: A text JSON file named model.json, which carries the topology and reference to the weights files, used by tensorflow.js.
```

Author: How is the weights file provided? Is it some URI that would need to be downloaded by the consumer?

Reviewer 2: Tensorflow.js models are not meant for downloading but served directly from a server. So we only need to point to the URL of the json file, which will then automatically link to weights by the model loader.

Author: Ok, thanks for the clarification.
